src/shared/utils/testUtils.ts

```diff
@@ -1,9 +1,55 @@
+import mockAxios, { HttpResponse } from 'jest-mock-axios';
```

Comment:
This is giving me an error causing the app to crash (blank page on `npm run start`)

![CleanShot 2024-04-15 at 14 02 24](https://github.com/ChildMindInstitute/mindlogger-admin/assets/2916156/9beba95b-6448-42be-bc17-9f89585542b2)

Removing this import removes the error. Probably related to removing `src/shared/tests/axios-mock.ts` as it was an excluded path

Reply:
Thanks, I got the PR in a working state similar to @farmerpaul's approach 👍